core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -1057,4 +1059,12 @@ public static void setFeedFilter(String value) {
                 .apply();
     }
 
+    public static long getUsageCountingDateMillis() {
+        return prefs.getLong(PREF_USAGE_COUNTING_DATE, -1);
+    }
+
+    public static void resetUsageCountingDate() {
```

Comment:
This should also be called when a database is imported.

Reply:
Thank you. I forgot this, but should not be the date set to `-1` (unset) because we do not know the date the new statistics were counted since?

Reply:
Oh, right. Sorry.